### PR TITLE
Clock done

### PR DESCRIPTION
**Pull request by illiarerun**
- [DEMO LINK](https://illiarerun.github.io/layout_stop-watch/)

### CODE REVIEW

**Review by lera-yaskevych — COMMENTED** [review]
The code is really good!
I guess I can only suggest you place the watch in the center of the page, just for the sake of practicing BEM and flexbox positioning, since there is no such requirement in the list, so it's up to you)

**Comment by illiarerun** [comment]
Thanks for your review! I should've thought about centering clock for better presentation)

File: src/styles/blocks/watch.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+$clock-size: 500px;
+
+.watch {
+  position: relative;
+  height: $clock-size;
+  width: $clock-size;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/watch.scss:6]: mixin square

**Review by Ihor-Karpyn — APPROVED** [review]
LGTM👌

File: src/styles/blocks/watch.scss (hunk 1 above)
Comment (illiarerun) [reply src/styles/blocks/watch.scss]: Done, should've thought about that)

**Pull request closed without merging**